src/widgets/ProfileWidget/subcomponents/ProfileHeader/ProfileHeader.scss

```
@@ -0,0 +1,24 @@
+@import "src/styles/index";
+
+.profile-header {
+  height: 9rem;
+  width: 100%;
+  // position: absolute;
+  display: flex;
+  flex-direction: column;
+  justify-content: end;
+  // left: 0;
+  // top: 0;
+  background-size: cover;
+  background-repeat: no-repeat;
```

```suggestion
  display: flex;
  flex-direction: column;
  justify-content: end;
  height: 9rem;
  width: 100%;
  background-size: cover;
  background-repeat: no-repeat;
```

https://github.com/airswap/airswap-marketplace/blob/main/CONTRIBUTING.md#order-of-properties